src/components/add-to-cart/add-to-cart.component.jsx

```diff
@@ -0,0 +1,44 @@
+/** @jsxImportSource @emotion/react */
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { addItemToCart } from '../../store/cart';
+
+import { CTA } from '../styles/button.styles';
+
+function AddToCart({ product, bubbles = true, ctaStyles = null }) {
+  const dispatch = useDispatch();
+
+  const { id, name, price, img } = product;
+
+  const handleAddToCart = (() => {
+    return bubbles
+      ? () => dispatch(addItemToCart({ id, name, price, img }))
```

Comment:
creo que aqui podemos usar la variable `product`
```suggestion
      ? () => dispatch(addItemToCart(product))
```